3_run_rsync/templates/pod.yml.j2

```diff
@@ -38,3 +38,22 @@ spec:
     env:
     - name: SSH_PUBLIC_KEY
       value: "{{ ssh_public_key['content'] | b64decode }}"
```

Comment:
You're missing the new command block for the rsync/ssh pod.

Reply:
see https://github.com/jmontleon/stunnel-ssh-poc/blob/master/ansible-poc/pod.yml.j2#L23-L26